evap/evaluation/migrations/0151_migrate_single_results_to_normal_evaluations.py

```diff
@@ -0,0 +1,60 @@
+from django.db import migrations
+
+
+def migrate_single_results(apps, _schema_editor):
+    Evaluation = apps.get_model("evaluation", "Evaluation")
+    Question = apps.get_model("evaluation", "Question")
+    Questionnaire = apps.get_model("evaluation", "Questionnaire")
+    RatingAnswerCounter = apps.get_model("evaluation", "RatingAnswerCounter")
+    TextAnswer = apps.get_model("evaluation", "TextAnswer")
+
+    # Don't create a questionnaire when running the migration on a database without single results
+    # E.g. when running on an empty database before loading test data
+    if not Evaluation.objects.filter(is_single_result=True).exists():
+        return
+
+    single_result_questionnaire = Questionnaire.objects.get(name_en="Single result")
+    assert single_result_questionnaire.questions.count() == 1
+    assert single_result_questionnaire.questions.first().type == 2  # GRADE question
+
+    overall_rating_questionnaire, created = Questionnaire.objects.get_or_create(
+        name_en="Total grade",
+        type=30,  # BOTTOM questionnaire
+    )
```

Comment:
We probably also want this, right?
```suggestion
    overall_rating_questionnaire, created = Questionnaire.objects.get_or_create(
        name_en="Total grade",
        type=30,  # BOTTOM questionnaire
        defaults={"name_de": "Gesamtnote"},
    )
```